docs/pipelines.md

```diff
@@ -6,27 +6,47 @@ weight: 3
 -->
 # Pipelines
 
-- [Overview](#pipelines)
-- [Configuring a `Pipeline`](#configuring-a-pipeline)
+- [Pipelines](#pipelines)
```

Comment:
i think something odd happened here - not sure why we have a top level link back to the top of the page? (maybe you're using a tool that's doing that?)

maybe these updates to the TOC would make sense in a separate PR?  i think i like having the more verbose TOC but it will be harder to keep up to date 🤔

Reply:
(feel free to ignore me)

Reply:
moved the TOC updates to a separate PR: https://github.com/tektoncd/pipeline/pull/3743